src/types/redis_set.cc

```diff
@@ -70,7 +70,11 @@ rocksdb::Status Set::Add(const Slice &user_key, const std::vector<Slice> &member
   WriteBatchLogData log_data(kRedisSet);
   batch->PutLogData(log_data.Encode());
   std::string sub_key;
+  std::unordered_set<std::string> mset;
```

Comment:
Would `set` or `unordered_set` for `Slice` be ok here?

Reply:
`rocksdb::Slice` is not ok, it can't pass the compiling.

Reply:
Yeah, seems it doesn't implement `std::hash`. Lets leave it for future optimizations